rocAL/source/augmentations/geometry_augmentations/node_slice.cpp

```diff
@@ -21,9 +21,7 @@ THE SOFTWARE.
 */
 
 #include "augmentations/geometry_augmentations/node_slice.h"
-
 #include <vx_ext_rpp.h>
```

Comment:
Why are we having these includes in .cpp file? Can we move it to .h?

Please make these changes in slice PR.